Beams/Testing/Cxx/vtkSlicerIECTransformLogicTest1.cxx

```diff
@@ -141,48 +141,91 @@ int vtkSlicerIECTransformLogicTest1(int vtkNotUsed(argc), char* vtkNotUsed(argv)
   if ( !IsTransformMatrixEqualTo(mrmlScene,
       iecLogic->GetTransformNodeBetween(vtkSlicerIECTransformLogic::FixedReference, vtkSlicerIECTransformLogic::RAS),
       expectedFixedReferenceToRasTransform_Translated_MatrixElements ) )
-    {
+  {
     std::cerr << __LINE__ << ": FixedReferenceToRasTransform transform does not match baseline for translated isocenter" << std::endl;
     return EXIT_FAILURE;
-    }
+  }
 
   double expectedBeamTransform_IsocenterTranslated_MatrixElements[16] =
     {  -1, 0, 0, 1000,   0, 0, 1, 200,   0, 1, 0, 0,   0, 0, 0, 1  };
   if ( !IsTransformMatrixEqualTo(mrmlScene,
       beamTransformNode, expectedBeamTransform_IsocenterTranslated_MatrixElements ) )
-    {
+  {
     std::cerr << __LINE__ << ": Beam transform does not match baseline for translated isocenter" << std::endl;
     return EXIT_FAILURE;
-    }
+  }
 
-  // Gantry angle, 1 degree
-  beamNode->SetGantryAngle(1.0);
+  // Gantry angle, "-1" degree

```

Comment:
No need for the quotes. Here or later ("90"), these are not strings but numbers.